File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/responses/LoginResponse.java
package com.eulersbridge.isegoria.network.api.responses;

import com.eulersbridge.isegoria.network.api.models.NewsArticle;
import com.eulersbridge.isegoria.network.api.models.User;

import java.util.List;

public class LoginResponse {

    public List<NewsArticle> articles;
    public User user;
    public long userId;

}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/responses/LoginResponse.kt
package com.eulersbridge.isegoria.network.api.responses

import com.eulersbridge.isegoria.network.api.models.NewsArticle
import com.eulersbridge.isegoria.network.api.models.User

data class LoginResponse (
    val articles: List<NewsArticle>?,
    val user: User?,
    val userId: Long
)
